bundles/org.openhab.binding.myq/README.md

```diff
@@ -31,7 +31,9 @@ Once an account has been added, garage doors and lamps will automatically be dis
 | Channel       | Item Type     | Thing Type       | States                                                 |
 |---------------|---------------|------------------|--------------------------------------------------------|
 | status        | String        | garagedoor       | opening, closed, closing, stopped, transition, unknown |
-| rollershutter | Rollershutter | garagedoor       | UP, DOWN, 0%, 100%                                     |
+| rollershutter | Rollershutter | garagedoor       | UP, DOWN, 0%, 100%     
+| closeError    | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)
+| openError     | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)|
 | switch        | Switch        | garagedoor, lamp | ON (open), OFF (closed)
```

Comment:
```suggestion
| rollershutter | Rollershutter | garagedoor       | UP, DOWN, 0%, 100%                                     |
| closeError    | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)               |
| openError     | Switch        | garagedoor       | ON (has error), OFF (doesn't have error)               |
| switch        | Switch        | garagedoor, lamp | ON (open), OFF (closed)                                
```

Reply:
FYI i use http://markdowntable.com/ to quickly fix tables like this...

Reply:
@digitaldan, thanks for the information. I will use this website next time 👍